.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -158,6 +158,12 @@ jobs:
         timeout-minutes: 15
         run: ./.github/scripts/run_e2e_tests.sh
 
+      - name: Print debug if failed
+        if: ${{ failure() }}
+        run: |
+          cd bin/cliain
+          cargo run -- --seed //Alice debug-storage
```

Comment:
just a nit: I prefer subcommand before flags, but feel free to ignore; resolve at your will

Reply:
wont work in that order